randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -1409,7 +1419,7 @@ Extra - map_name: rm_a3b07
   * Layers: default
   * Event Metroids Area 3 - Breeding Grounds Middle Top Gamma
   > Tunnel to Breeding Grounds Alpha Nest
-      Trivial
+      Can Use Any Bombs and Hijump Tunnel Climb
```

Comment:
is this actually reachable with just hijump?

Reply:
Yes it is possible with HJ

Reply:
all right then